tests/Unit/Suites/ClientTest.php

```diff
@@ -126,7 +127,7 @@ public function testItAddsChoicesMapping(): void
     public function testItThrowsAnExceptionIfFieldDoesNotExist(): void
     {
         $this->expectException(ClientException::class);
-        $this->expectErrorMessage('Unrecognized field name "non-existing-field-name"');
+        $this->expectExceptionMessage('Unrecognized field name "non-existing-field-name"');
```

Comment:
`exoectErrorMessage` is deprecated, and will be removed in PHPUnit 10. For our purposes `expectExceptionMessage` should work the same.